### DISCUSSION TITLE
Discrepancies with server side tracking

### DISCUSSION THREAD

**Discussion by TkDodo** [plausible/analytics#1283] [Anything Else]
I’m seeing some pretty large discrepancies between netlify analytics and plausible, e.g. for yesterday:

pageviews: netlify 2726 / plausible 1383
unique visitors: netlify 1170 / plausible 687

only explanation I could see is that 50% of my users use some tracking blockers or browsers like brave, but that seems like a pretty large number. Are there any other explanations?

**Comment by metmarkosaric** [comment]
There are 2 parts to this:

1. Normally server side tracking doesn't exclude bots - do you know if Netlify analytics excludes bots? Plausible excludes bots, crawlers and all the other known non-human activity by default. There's a comparison here in analytics between Plausible and server side tools https://plausible.io/blog/server-log-analysis

2. Do you use our default script or the proxy? You should set up our Netlify proxy if you're worried about adblockers. Then your Plausible script will only be blocked by those people who completely disable JavaScript, while it will not be blocked by any browser or adblocker. See https://plausible.io/docs/proxy/guides/netlify

**Reply by TkDodo** [reply]
I'm using the default script. Thanks, I'll take a look at the proxy solution

**Reply by TkDodo** [reply]
Alright, I just shipped the proxy solution. Using brave, I see that I am now also tracked properly 👍 .

What confused me a bit was that also numbers of users that had a referrer like `google` also were a lot lower in numbers, and I don't think that bots have referrers? (or maybe they do 🤷 )

**Reply by metmarkosaric** [reply]
great, looking forward to seeing how it looks now! 

but yeah i can confirm that Netlify does NOT exclude bot traffic from their analytics so their stats are not that reliable for web analytics purposes but more for server log purposes

"We include bot traffic; google doesn’t."
see https://answers.netlify.com/t/over-100x-more-views-in-netlify-analytics-than-google-analytics/15403/2

**Comment by ukutaht** [comment]
What @metmarkosaric said. In a test he ran we saw 18x more page views with server side tracking vs client-side tracking. Most of the difference comes from the fact that client-side analytics (Plausible) only counts visitors that run Javascript. It's a decent proxy for '*this is probably a real human using a web browser*'.

Server logs will include all HTTP requests regardless of whether they came from a normal browser, headless browser, `curl`, Google Bot, etc. Depending on your site there will be a lot of bot traffic in your server logs. For us the difference was 18x
